learn-more.css

```diff
@@ -0,0 +1,311 @@
+
+body {
+    font-family: avenir; 
+}
+button {
+    display: inline-block;
+    text-align: center;
+    height: 35px;
+    width: 70px;
+    background-color: #197278;
+    color: white;
+    font-size: 10px; 
+    border-radius: 3px;
+    font-weight: bold; 
+    text-transform: uppercase;
+}
+button:hover {
+    opacity: 50%;
+}
+
+.margin {
+    left: 0in; 
+    right: 0in; 
+}
+
+.border-below {
+    border-bottom: 5px double black; 
+    padding-bottom: 2vw;
+    margin-bottom: 2vw; 
+}
+
+/*-----------------HEADER---------------------*/
+#header {
+    display: block;
+    text-align: center;
+    text-decoration: none;
+}
+#header h1 {
+    font-size: 4em;
+    padding: 0;
+    margin-bottom:0;
+    grid-column:1;
+    grid-row: 1; 
+    text-decoration: none;
+}
+a:link {
+    text-decoration: none;
+}
+
+#header h2 {
+    padding: 1vw; 
+    margin: 0; 
+}
+#header h3 {
+    grid-column: 1;
+    grid-row: 2;
+    padding-bottom: 1in; 
+    margin-top: 0;
+    margin-bottom: 5vw;
+    border-bottom: 5px darkcyan double;
+}
+.cr-button {
+    margin: 1vw;
+    width: 10vw; 
+}
+
+/*Container to keep images in line*/
+.image-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr; 
+    grid-template-rows: 4fr auto; 
+    grid-gap: 1em;
+    padding-top: 1vw; 
+}
+#image1 {
+    grid-column:1;
+    grid-row: 1; 
+    object-fit: cover;
+    max-width: 100%;
+    max-height: 100%; 
+    -webkit-box-shadow: 0 8px 6px -6px black;
+    -moz-box-shadow: 0 8px 6px -6px black;
+         box-shadow: 0 8px 6px -6px black;
+}
+#image2 {
+    grid-column:2;
+    grid-row: 1; 
+    object-fit: cover;
+    max-width: 100%;
+    max-height: 100%;
+    -webkit-box-shadow: 0 8px 6px -6px black;
+    -moz-box-shadow: 0 8px 6px -6px black;
+         box-shadow: 0 8px 6px -6px black;
+}
+#image3 {
+    grid-column:3;
+    grid-row: 1; 
+    object-fit: cover;
+    max-width: 100%;
+    max-height: 100%; 
+    -webkit-box-shadow: 0 8px 6px -6px black;
+    -moz-box-shadow: 0 8px 6px -6px black;
+         box-shadow: 0 8px 6px -6px black;
+}
+
+#quote {
+    font-size: 2em; 
+    background-image: url(https://image.freepik.com/free-photo/light-blue-gradient-abstract-banner-background_8087-1851.jpg);
+    background-size: 100%;
+    color: white; 
+    display: block;
+    margin-left: 0;
+    margin-right: 0;
+    text-align: center;
+    padding: 2vw 15vw;
+}
+/*-----------------MAIN SECTION---------------------*/
+/*grid to hold two columns*/ 
+#home {
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: 1fr minmax(5em, auto);
+    grid-gap: 1em;
+}
+
+/*left side more in depth text*/
+.container1 {
+    grid-column: 2;
+    display: flex;
+    flex-direction: column;
+    padding: 2px; 
+}
+.large-image {
+    object-fit: cover;
+    max-width: 100%;
+    max-height: 100%;
+	-webkit-box-shadow: 0 8px 6px -6px black;
+	   -moz-box-shadow: 0 8px 6px -6px black;
+	        box-shadow: 0 8px 6px -6px black;
```

Comment:
box shadow is one of my favorite CSS attributes. Also nice work covering different internet browsers to make sure your box shadow is handled across browsers